console/console-init/ui/src/components/AdapterList/AdapterList.tsx

```diff
@@ -49,7 +49,7 @@ const Adapter: React.FC<{ adapter: IAdapter }> = ({ adapter }) => {
   return (
     <>
       <Title size="xl" headingLevel="h2">
-        {type.toUpperCase() + " Adapter"}
+        {type && type.toUpperCase() + " Adapter"}
```

Comment:
we can avoid for undefined check with && and replace the logic of `type && type.toUpperCase()` with `type?.toUpperCase()`